.circleci/ci/lighthouse.js

```diff
@@ -0,0 +1,161 @@
+/*******************************************************************************
+ * Copyright 2023 Adobe
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ******************************************************************************/
+
+const fs = require('fs');
+const https = require('https');
+const ci = new (require('./ci.js'))();
+
+
+const checkLightHouse = async () => {
+    const lighthouse = await import('lighthouse')
+    const chromeLauncher = await import('chrome-launcher')
+    const chrome = await chromeLauncher.launch({chromeFlags: ['--headless']});
+    const options = {logLevel: 'info', output: 'html', port: chrome.port, extraHeaders: { Authorization: 'Basic ' + Buffer.from('admin:admin').toString('base64') }};  // YWRtaW46YWRtaW4= -- base64 encoded, admin:admin
+
+    const lighthouseConfig = JSON.parse(fs.readFileSync('/home/circleci/build/.circleci/ci/lighthouseConfig.json'))
+
+    const runnerResult = await lighthouse.default(lighthouseConfig.urls[0], options);
+    // `.report` is the HTML report as a string
+
+    const reportHtml = runnerResult.report;
+    console.log('Report is done for', runnerResult.lhr.finalDisplayedUrl);
+    console.log(getCommentText(runnerResult.lhr.categories))
+
+    //postCommentToGitHub(getCommentText(runnerResult.lhr.categories), process.env.GITHUB_TOKEN)
```

Comment:
Why is this commented ? Please enable the same

Reply:
commented, as we don`t have the adobe-bot token, so commented it, will enable, once we get the token, should i remove the method for now?

Reply:
created a JIRA ticket for the same https://jira.corp.adobe.com/browse/FORMS-8959
once we are clear with the token thing, I will enable the same.